hack/rpms.sh

```diff
@@ -76,16 +77,18 @@ function createrepo() {
 }
 
 if [ "${VALIDATE}" != "1" ]; then
-    if ! [ -f "${BUILDDIR}/security/hpe-signing-key.asc" ]; then
-        echo "Downloading HPE signing key"
-        mkdir -p "${BUILDDIR}/security"
-        acurl -Ss -o "${BUILDDIR}/security/hpe-signing-key.asc" "${HPE_SIGNING_KEY}"
-    fi
-    if ! [ -f "${BUILDDIR}/security/hpe-signing-key-fips.asc" ]; then
-        echo "Downloading new HPE signing key"
-        mkdir -p "${BUILDDIR}/security"
-        acurl -Ss -o "${BUILDDIR}/security/hpe-signing-key-fips.asc" "${HPE_SIGNING_KEY_FIPS}"
-    fi
+    # Download and store RPM signing keys.
```

Comment:
I think this is missing `docs-csm` processing piece:
https://github.com/Cray-HPE/csm/blob/release/1.6/hack/rpms.sh#L80-L88